CalculatorInput/CalculatorInputView.m

```diff
@@ -11,6 +11,7 @@ @interface CalculatorInputView ()
 @implementation CalculatorInputView
 
 - (id)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
     self = [[[NSBundle bundleForClass:[self class]] loadNibNamed:@"CalculatorInputView" owner:self options:nil] firstObject];
```

Comment:
this bugs me … but 🆗

Reply:
I can't suppress the warning and it kills me that it leaks into the app. I think the real problem is that this loading-from-a-nib-in-`-initWithFrame:` thing is a total hack. This should be totally fine since we're re-assigning to self on the next line?